### PR TITLE
File Attachment Runtime

### PR DESCRIPTION
**Pull request by rismehta**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by rismehta** [comment]
Waiting for model changes to get released, right now this PR would fail

File: ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                    expect(afFormData.contentType).to.equal("application/json");
+                    assert.equal(afFormData.attachments.length, 2, "incorrect attachments returned from guideBridge#getFormDataObject API");
+                    afFormData.attachments.forEach(function(attachment) {
+                        expect(attachment.dataRef).to.contain('fileinput');
+                    });
+                    // explicitly calling this to ensure there are no errors during this API invocation
+                    let htmlFormData = afFormData.toHTMLFormData();
+                    // Display the key/value pairs
+                    for (let pair of htmlFormData.entries()) {
+                        console.log(pair[0]+ ', ' + pair[1]);
```
Comment (nit23uec) [inline ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js:110]: is there any use of doing console.log in the tests, why not assert it?

File: ui.frontend/src/index.js [hunk 2]
```diff
@@ -38,4 +38,4 @@ const Actions = {
     Click, Change, Submit, Blur, AddItem, RemoveItem
 }
```
Comment (nit23uec) [inline ui.frontend/src/index.js:40]: we should create a wrapper for File Attachment related APIs. it becomes easy for developer to understand and use.
```
const FileAttachmentUtils = {
 FileObject, extractFileInfo
}
export {...., FileAttachmentUtils}
```

File: ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js (hunk 1 above)
Comment (rismehta) [reply ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js:110]: We can't assert, I have added a comment, on why we do console log. This is just to ensure there is no error

**Review by barshat7 — APPROVED** [review]

**Pull request merged**